bin/commands/help/index.ts

```diff
@@ -0,0 +1,38 @@
+/*
+  This program and the accompanying materials are made available
+  under the terms of the Eclipse Public License v2.0 which
+  accompanies this distribution, and is available at
+  https://www.eclipse.org/legal/epl-v20.html
+ 
+  SPDX-License-Identifier: EPL-2.0
+ 
+  Copyright Contributors to the Zowe Project.
+*/
+
+import * as std from 'cm_std';
+import * as common from '../../libs/common';
+
+export function execute() {
+  const errorCode: string = std.getenv('ZWE_CLI_PARAMETER_ERROR_CODE');
+
+  const serverCode: string = errorCode.charAt(3);
+
+  if (/^[zZ][wW][eE][AaSsDdLl][A-Za-z]?[0-9]{3,4}[A-Za-z]$/.test(errorCode)) {
+    if (serverCode.toLowerCase() === 'd') {
+      common.printMessage("This code corresponds to the errors related to the ZOWE Desktop and the App Server.");
+      common.printMessage("To find the description of this error code, refer to the Zowe documentation at https://github.com/zowe/docs-site/blob/master/docs/troubleshoot/app-framework/appserver-error-codes.md");
+    } else if (serverCode.toLowerCase() === 's') {
+      common.printMessage("This code corresponds to the errors related to the Zowe Subsystem Services (ZSS) and Zowe Installation Services (ZIS)");
+      common.printMessage("To find the description of this error code, refer to the Zowe documentation for ZSS at https://docs.zowe.org/stable/troubleshoot/app-framework/zss-error-codes and for ZIS at https://github.com/zowe/docs-site/blob/master/docs/troubleshoot/app-framework/zis-error-codes.md");
+      common.printMessage(`You may also explore reports from other users experiencing the same error by searching here https://github.com/search?q=org%3Azowe+${errorCode}&type=discussions`)
```

Comment:
Move this
>       common.printMessage(`You may also explore reports from other users experiencing the same error by searching here https://github.com/search?q=org%3Azowe+${errorCode}&type=discussions`)

This is true for ALL error codes. Or at least, any that start with "ZWE"! Currently you only have it for zweS. I don't want people to lose out on this tip.